integration-tests/cli/product-json-to-csv.it.js

```diff
@@ -80,6 +85,10 @@ describe('CSV and CLI Tests', () => {
       typeId: 'channel',
       id: priceChannel[0].body.id,
     }
+    sampleProducts[0].masterVariant.prices[0].customerGroup = {
```

Comment:
why wouldn't you move this logic into `createProducts` and avoid mutation (during tests)?
mutation during test could lead to harmful false positive

Reply:
That's a great question. It looks like the initial code that I followed was a bit sloppy. Moved both the mutation of `channel` and `customerGroup` into this function instead
https://github.com/commercetools/nodejs/blob/b67baed9170c60674007723b3d06e6a9570e0277/integration-tests/cli/product-json-to-csv.it.js#L83

in this commit: https://github.com/commercetools/nodejs/pull/1168/commits/d18dc1ea926ebc183f1977eb8b65ddb7577d69b2